app/lib/features/space/pages/events_page.dart

```diff
@@ -24,164 +23,161 @@ class SpaceEventsPage extends ConsumerWidget {
   Widget build(BuildContext context, WidgetRef ref) {
     final upcoming = ref.watch(spaceUpcomingEventsProvider(spaceIdOrAlias));
     final past = ref.watch(spacePastEventsProvider(spaceIdOrAlias));
-    return DecoratedBox(
```

Comment:
I trust that this hasn't changed in inner code and the diff is just because of the change of indentation, so I skipped over it.

Reply:
This change is just about removing gradient decoration.